server/src/main/java/com/distkv/server/storeserver/runtime/workerpool/Worker.java

```diff
@@ -143,19 +143,17 @@ private void syncToSlaves(DistkvRequest request, CompletableFuture<DistkvRespons
   }
 
   // Check if it's a request with ttl.
-  private static boolean isTimeRemainingQuery(DistkvRequest distkvRequest) {
+  private static boolean isTTLRequest(DistkvRequest distkvRequest) {
     RequestType requestType = distkvRequest.getRequestType();
     return requestType == TTL;
   }
 
   //
-  private void isExistsInStore(long survivalTime, String key) {
-    if (survivalTime == -1) {
-      boolean exists = storeEngine.exists(key);
-      if (!exists) {
-        throw new KeyNotFoundException("The key {1} not found in store.", key);
-      }
+  private boolean checkExistsInStore(long timeToLive, String key) {
```

Comment:
Since we do not throw any exception, there is no need to name it `checkXXX`.  Just use existsInStore instead.